backend/drm/drm.c

```diff
@@ -227,13 +232,26 @@ static void wlr_drm_connector_set_gamma(struct wlr_output *output,
 		uint32_t size, uint16_t *r, uint16_t *g, uint16_t *b) {
 	struct wlr_drm_connector *conn = (struct wlr_drm_connector *)output;
 	struct wlr_drm_backend *drm = (struct wlr_drm_backend *)output->backend;
-	drmModeCrtcSetGamma(drm->fd, conn->crtc->id, size, r, g, b);
+	bool ok;
```

Comment:
Nitpick: we're not in the 89s anymore, we can define variables when needed instead of at the beginning of functions.